tests/behat/features/asp/outages-unauth.feature

```diff
@@ -21,11 +21,15 @@ Feature: test ACCESS Support Outages Page
     Then I should see "Planned Downtimes"
     Then I should see "All Outages"
     # TODO Figure out how to add below test
-    # Then I should see "Show 10 Entries"
+    Then I should see "show"
+    Then I should see "entries"
+    #outages-planned_length needs testing
     # TODO search doesn't seem to be working
-    # When I fill in "Search" with "ACCESS"
-    # Then I should see "ACCESS Metrics XDMoD Update"
-    # When I fill in "Search" with ""
-    # Then I should see "Showing 1 to 1 of 1 entries"
-    # TODO Paginitation test does not work
-    # When I click "2"
+     When I fill in "Search" with "ACCESS"
+     And I wait 5 seconds
+     Then I should see "ACCESS User Identity"
+     When I fill in "Search" with ""
+     And I wait 5 seconds
+    Then I should see "Showing 1 to 2 of 2 entries"
```

Comment:
This is testing "Planned Downtimes" which will change regularly. We shouldn't include a behat test for the number of entries in pagination. 

Let's change this test to search for a specific outage, load the page for that outage, and test that the details are visible.